src/tests/EdFi.DataManagementService.Tests.E2E/Features/Descriptors/CreateDescriptorsValidation.feature

```diff
@@ -116,15 +112,14 @@ Feature: Descriptors are a set of mechanisms to support flexible enumerations or
               And the response body is
                   """
                     {
-                        "detail": "Access to the resource could not be authorized. The 'Namespace' value of the resource does not start with any of the caller's associated namespace prefixes ('uri://ed-fi.org', 'uri://gbisd.org', 'uri://tpdm.ed-fi.org').",
+                        "detail": "Access to the resource could not be authorized. The 'Namespace' value of the resource does not start with any of the caller's associated namespace prefixes ('uri://ed-fi.org').",
```

Comment:
Although we are ignoring this test right now, I simplified in advance for when we get to it. We probably won't setup namespace security with multiple namespaces.